.pre-commit-config.yaml

```diff
@@ -19,6 +19,7 @@ default_stages: [commit, push]
 default_language_version:
   # force all unspecified python hooks to run python3
   python: python3
+  node: 16.14.2
```

Comment:
I don't have as much context, but I'm curious: what's the reason to go with 16.14.2?
It looks like 17 has basically the same support for the same set of platforms/architectures ([source](https://github.com/nodejs/node/blob/v17.x/BUILDING.md#platform-list)). v18 is where the troubles seems to start from what I can tell by the docs and the comments in the linked Githu Issue ([source](https://github.com/nodejs/node/blob/main/BUILDING.md#platform-list)).

Reply:
16 is an LTS release, so that makes sense to me. Why aren't we on the latest though, 16.5.1?

Reply:
Yeah. 16.15.1 is better (unless it causes the problem) - but I think it's 18.* that causes the problem for you ?

Reply:
@o-nikolas You're right, even version 17 works fine. I agree @jedcunningham, @potiuk it makes more sense to use the latest version for v16.* . It is v18.* that causes the problem.